R/get_schema.R

```diff
@@ -40,10 +40,15 @@ get_schema.SQLiteConnection <- function(.x) {
   } else if ("temp" %in% schemata) {
     return("temp")
   } else {
-    return()
+    return(NA_character_)
```

Comment:
Consider if the outputs of the other `get_schema` should also be `NA_character_` if schema is unset. 
Now I don't think the "empty" outputs of the different `get_schema` methods are not aligned. `NA_character_` vs `NULL`